### PR TITLE
[interp] Add more netcore intrinsics.

### PR DESCRIPTION
**Pull request by vargaz**

### CODE REVIEW

File: mono/mini/interp/transform.c [hunk 1]
```diff
@@ -1217,6 +1217,35 @@ interp_handle_intrinsics (TransformData *td, MonoMethod *target_method, MonoMeth
...
+			int val;
+
+			t = mini_get_underlying_type (arg_type);
+
+			if (MONO_TYPE_IS_PRIMITIVE (t) && t->type != MONO_TYPE_R4 && t->type != MONO_TYPE_R8)
+				val = 1;
+			else
+				val = 0;
+			interp_add_ins (td, MINT_LDC_I4);
+			WRITE32_INS (td->last_ins, 0, &val);
```
Comment (lewurm) [inline mono/mini/interp/transform.c]: there is `MINT_LDC_I4_0` and `MINT_LDC_I4_1`

File: mono/mini/interp/transform.c [hunk 2]
```diff
@@ -1217,6 +1217,35 @@ interp_handle_intrinsics (TransformData *td, MonoMethod *target_method, MonoMeth
 			PUSH_SIMPLE_TYPE (td, STACK_TYPE_I4);
 			td->ip += 5;
 			return TRUE;
+		} else if (!strcmp (tm, "AreSame")) {
+			*op = MINT_CEQ_I4 + STACK_TYPE_I - STACK_TYPE_I4;
```
Comment (lewurm) [inline mono/mini/interp/transform.c]: there is `MINT_CEQ_P`

**Comment by vargaz** [comment]
@monojenkins build failed

**Comment by vargaz** [comment]
@monojenkins build failed

**Pull request merged**